doc/faq.markdown

```diff
@@ -1,11 +1,23 @@
 Frequently Asked Questions {#faq}
 ==========================
 
--   Q: Example question?
+-   **What is InputArray?**
 
-    A: Example answer
+        It can be seen that almost all OpenCV functions receive InputArray type.
+        What is it, and how can I understand the actual input types of parameters?
 
 
--   Q: Example question?
+    This is the proxy class for passing read-only input arrays into OpenCV functions.
 
-    A: Example answer
+    Inside a function you should use `_InputArray::getMat()` method to construct
```

Comment:
Please, use `cv::` here, because now doxygen generates a reference to `cv::hpp::_InputArray::getMat()`